CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [v0.1.2](https://github.com/odpf/raccoon/compare/v0.1.1...v0.1.2) (2021-11-23)
+
+
+### Features
+
+* Refactor codebase to Collector based design ([#19 ](https://github.com/odpf/raccoon/issues/19)) ([193ba4d](https://github.com/odpf/raccoon/commit/193ba4d68fd2ee41fe05acde11ee6fdc155fdaee ))
+* Added support for JSON in websocket ([#19 ](https://github.com/odpf/raccoon/issues/19)) ([193ba4d](https://github.com/odpf/raccoon/commit/193ba4d68fd2ee41fe05acde11ee6fdc155fdaee ))
+* Added support for REST endpoint with JSON and protobuf as supported data formats.  ([#19 ](https://github.com/odpf/raccoon/issues/19)) ([193ba4d](https://github.com/odpf/raccoon/commit/193ba4d68fd2ee41fe05acde11ee6fdc155fdaee )) 
+* Added support for GRPC. ([#19 ](https://github.com/odpf/raccoon/issues/19)) ([193ba4d](https://github.com/odpf/raccoon/commit/193ba4d68fd2ee41fe05acde11ee6fdc155fdaee ))
+
+
+### Bug Fixes
+
+NA
+
```

Comment:
How was the changelog on v0.1.2 GH release generated?

Reply:
Did it directly from the UI.